server/lease/lessor.go

```diff
@@ -821,6 +830,7 @@ func (le *lessor) initAndRecover() {
 		}
 	}
 	le.leaseExpiredNotifier.Init()
+	initLeaseCount.Add(float64(len(lpbs)))
```

Comment:
I think the init can be called multiple times during server lifecycle.

Reply:
My understanding was that in reverse calling stack, this is only ever initiated once per etcd member, but I might've missed something. `NewServer` -> `NewLessor` -> `newLessor` -> `l.initAndRecover`?

The intuition behind this metric is to see how many leases are available at the start. An issue recently occurred where we accumulated a lot of leases, and we wanted to get a number, so that it gives us an approximate idea on how long it will take for these leases to be revoked.

@serathius

Reply:
@serathius any feedback here as well?^